src/app/[locale]/dataset-list.cy.tsx

```diff
@@ -1,36 +1,72 @@
-import DatasetList from './dataset-list';
+import DatasetList, {Props} from './dataset-list';
+import {NextIntlClientProvider} from 'next-intl';
+import messages from '@/messages/en.json';
 
-describe('<DatasetList />', () => {
-  context('List rendering', () => {
-    it('Shows 1 dataset card if one dataset is returned in the search result', () => {
-      cy.fixture('search-results/one-dataset.json').then(searchResult => {
-        cy.intercept({method: 'GET', url: '/api/datasets*'}, searchResult);
-        cy.mount(
-          <DatasetList initialSearchResult={searchResult} locale="en" />
-        );
-        cy.getBySel('dataset-card').its('length').should('eq', 1);
-      });
+function DatasetListWithTranslation(props: Props) {
+  return (
+    <NextIntlClientProvider locale="en" messages={messages}>
+      <DatasetList {...props} />
+    </NextIntlClientProvider>
+  );
+}
+
+describe('List rendering', () => {
+  it('Shows 1 dataset card if one dataset is returned in the search result', () => {
+    cy.fixture('search-results/one-dataset.json').then(searchResult => {
+      cy.intercept({method: 'GET', url: '/api/datasets*'}, searchResult);
+      cy.mount(
+        <DatasetListWithTranslation
+          initialSearchResult={searchResult}
+          locale="en"
+        />
+      );
+      cy.getBySel('dataset-card').its('length').should('eq', 1);
+    });
+  });
+
+  it('Shows 3 dataset cards if 3 datasets are returned in the search result', () => {
+    cy.fixture('search-results/multiple-datasets.json').then(searchResult => {
```

Comment:
Minor: this test explicitly tests for 'three datasets', not for 'multiple datasets'. Should we rename `multiple-datasets.json` to `three-datasets.json`, to be clear about it content?